### PR TITLE
Add COVID-19 information to workshop reminder emails

### PR DESCRIPTION
**Pull request by islemaster**
[PLC-798](https://codedotorg.atlassian.net/browse/PLC-798) Adds COVID-19 copy to our workshop reminder emails, helping teachers and regional partners prepare appropriately.

Here's the new copy, as rendered by our mailer preview system:

image

Copy was drafted [in this doc (internal)](https://docs.google.com/document/d/1nEe2fW1FkoVoJTV0oaKvQ7DqrgzlOdWUHXgazFW3yHE/edit#).

This new copy appears in the following WorkshopMailer templates, for all workshop types except Admin/Counselor:

- Teacher-facing:
  - teacher_enrollment_receipt
  - teacher_enrollment_reminder
  - detail_change_notification
- Facilitator-facing
  - facilitator_enrollment_reminder
  - facilitator_detail_change_notification
- Organizer-facing
  - organizer_enrollment_reminder
  - organizer_detail_change_notification

## Testing story

Tested manually using mailer previews at `/rails/mailers/pd/workshop_mailer`.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by molly-moen — APPROVED** [review]

File: dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+%h3 COVID-19 precautions
+
+%p
+  In light of the global situation surrounding the COVID-19 virus, we understand that traveling and
+  gathering for a workshop may be cause for anxiety or concern for teachers registered for upcoming
+  Code.org workshops. We encourage you to refer to the
+  = link_to 'Centers for Disease Control and Prevention', 'https://www.cdc.gov/', target: '_blank'
+  (CDC) for the most up to date recommendations, as is your Regional Partner in evaluating whether
```
Comment (breville) [inline dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml]: nit: up-to-date

File: dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  to host your workshops as planned. If you have returned from a
+  = link_to 'Level 3 country', 'https://www.cdc.gov/coronavirus/2019-ncov/travelers/index.html', target: '_blank'
+  within the past 14-days, are feeling ill or show any flu-like symptoms such as a fever or cough,
```
Comment (breville) [inline dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml]: nit: oxford comma before "or"?

File: dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  please inform your Regional Partner.
+
+%p
+  Our workshops are highly interactive. At time of this update, the CDC suggests all individuals
```
Comment (breville) [inline dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml]: nit: At the time

File: dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  %ul
+    %li
+      Avoid contact with sick people.
+    %li
+      Avoid touching your eyes, nose, or mouth with unwashed hands.
+    %li
+      Clean your hands often by washing them with soap and water for at least 20 seconds or using
+      an alcohol-based hand sanitizer that contains at least 60% alcohol.
+    %li
+      It is especially important to clean hands after going to the bathroom; before eating; and
```
Comment (breville) [inline dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml]: If we're using semicolons, suggest:

It is especially important to clean hands: after going to the bathroom; before eating; and after coughing, sneezing, or blowing your nose.

File: dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+%h3 COVID-19 precautions
+
+%p
+  In light of the global situation surrounding the COVID-19 virus, we understand that traveling and
+  gathering for a workshop may be cause for anxiety or concern for teachers registered for upcoming
+  Code.org workshops. We encourage you to refer to the
+  = link_to 'Centers for Disease Control and Prevention', 'https://www.cdc.gov/', target: '_blank'
+  (CDC) for the most up to date recommendations, as is your Regional Partner in evaluating whether
+  to host your workshops as planned. If you have returned from a
```
Comment (breville) [inline dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml]: "as is" doesn't read right.  Suggest something like:

We encourage you to refer to the Centers for Disease Control and Prevent (CDC) for the most up-to-date recommendations.  Your regional partner will do the same when evaluating whether to host your workshops as planned.

**Comment by islemaster** [comment]
@breville Accepted all of your suggestions, and updated the PR description with a new screenshot of the mailer preview of this copy.  PTAL?

**Review by breville — APPROVED** [review]

File: dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  %ul
+    %li
+      Avoid contact with sick people.
+    %li
+      Avoid touching your eyes, nose, or mouth with unwashed hands.
+    %li
+      Clean your hands often by washing them with soap and water for at least 20 seconds or using
+      an alcohol-based hand sanitizer that contains at least 60% alcohol.
+    %li
+      It is especially important to clean hands: After going to the bathroom; before eating; and
```
Comment (breville) [inline dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml]: super nit, not sure if After needs big A. 

:)

Comment (islemaster) [reply dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml]: :face_with_head_bandage: Adding this to the list of elements of style that I wasn't taught comprehensively.

**Pull request merged**